app/models/posts.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author
```

Comment:
- [ ] The same happens with this one, since by default Rails looks for a table named `Authors`, we need to tell it that this is just an alias for table `Users`:

```suggestion
  belongs_to :author, class_name: 'User'
```

Please consider applying this! 🤓 🔧

Reply:
Thank you for your kind feedback, working on it @anagudelogu.